packages/server/lib/browsers/chrome.ts

```diff
@@ -450,16 +570,21 @@ export = {
     ])
 
     await this._navigateUsingCRI(pageCriClient, options.url)
+
+    if (options.experimentalSessionAndOrigin) {
```

Comment:
Since we have a bit of a different process now for attaching to target and getting the browser CRI client, we need to make sure that the appropriate even listeners are applied every time we attach to a new target, as well as turn on any necessary CDP events.